core/server/src/main/java/alluxio/master/file/FileSystemMaster.java

```diff
@@ -479,19 +479,20 @@ public Status getStatus() {
   /**
    * @return the status of the startup consistency check and inconsistent paths if it is complete
    */
-  public StartupConsistencyCheck getStartupConsistencyCheck() {
+  public StartupConsistencyCheckResult getStartupConsistencyCheck() {
     if (!Configuration.getBoolean(PropertyKey.MASTER_STARTUP_CONSISTENCY_CHECK_ENABLED)) {
-      return new StartupConsistencyCheck(StartupConsistencyCheck.Status.DISABLED, null);
+      return new StartupConsistencyCheckResult(StartupConsistencyCheckResult.Status.DISABLED, null);
     }
     if (!mStartupConsistencyCheck.isDone()) {
-      return new StartupConsistencyCheck(StartupConsistencyCheck.Status.RUNNING, null);
+      return new StartupConsistencyCheckResult(StartupConsistencyCheckResult.Status.RUNNING, null);
```

Comment:
I'm suggesting methods in `StartupConsistencyCheckResult` like
```java
public static StartupConsistencyCheckResult Failure() {
  return StartupConsistencyCheckResult(Status.FAILURE, null);
}

public static StartupConsistencyCheckResult Success(List<AlluxioURI> inconsistentUris) {
  return new StartupConsistencyCheckResult(Status.SUCCESS, inconsistentUrls);
}
```

and then we can make the constructor private and the call sites become nicer